### ISSUE TITLE
@content-build-action-not-suported

### ISSUE THREAD

**Issue by shainefisher** [Fody/Fody#495]
FodyWeavers.xml in the Android project throws this error on build when usingv2.5.0.280555 Xamarin.Forms release.
The fix is to set the build action for FodyWeavers.xml to none, but I think that may cause an issue and wanted to see what the plan going forward is in order to prevent this error?

Thanks

**Comment by SimonCropp** [comment]
yes build type should be none. but nuget does not allow this to be done in a package. so it is up to you. 

> FodyWeavers.xml in the Android project throws this error

sounds like a bug in android?
